ember-cli-build.js

```diff
@@ -1,12 +1,12 @@
-/* global require, module */
+/* eslint-env node */
 var path = require('path');
-var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+'use strict';
```

Comment:
I think this needs to happen before any other expression

Reply:
yes, confirm